lib/charms/sdcore_upf_k8s/v0/fiveg_n3.py

```diff
@@ -216,16 +216,16 @@ class N3AvailableEvent(EventBase):
     """Dataclass for the `fiveg_n3` available event."""
 
     def __init__(self, handle, upf_ip_address: str):
-        """Sets certificate."""
+        """Set certificate."""
         super().__init__(handle)
         self.upf_ip_address = upf_ip_address
 
     def snapshot(self) -> dict:
-        """Returns event data."""
+        """Return event data."""
         return {"upf_ip_address": self.upf_ip_address}
 
     def restore(self, snapshot):
-        """Restores event data."""
+        """Restore event data."""
```

Comment:
I assume that's because of the new tool, correct?

Reply:
yes, indeed